package.json

```diff
@@ -49,7 +49,7 @@
     "clean": "rm -rf lib/ dist/ coverage/ .nyc_output/ jsdoc/ test/e2e/.soroban",
     "docs": "yarn build:docs && jsdoc -c ./config/.jsdoc.json --verbose",
     "test": "yarn build:test && yarn test:node && yarn test:integration && yarn test:browser",
-    "test:e2e": "./test/e2e/initialize.sh && ava",
+    "test:e2e": "./test/e2e/initialize.sh && yarn _nyc mocha --recursive 'test/e2e/src/test-*.js'",
```

Comment:
Curious if you tried this

Reply:
`✖ ERROR: --parallel runs test files in a non-deterministic order, and is mutually exclusive with --sort`

I think it might be due to the usage of `beforeAll`